qbft/spectest/comparable/types.go

```diff
@@ -2,33 +2,36 @@ package qbftcomparable
 
 import (
 	"encoding/hex"
+
 	"github.com/bloxapp/ssv-spec/qbft"
 )
 
-var RootRegister = map[string]*qbft.State{}
+var RootRegister = map[string]*qbft.RootGetter{}
 
 type StateComparison struct {
-	PostState *qbft.State
+	qbft.RootGetter
```

Comment:
Make it a pointer like Alon did.
Maybe also give the variable some meaningful name?

`PostTestedObjectState` or something (it is kind of long, other ideas are welcome)? 

Also can you explain to me why we can't have a general StateComparison? Why must it be under QBFT?